gr-uhd/grc/uhd_rfnoc_graph.block.yml

```diff
@@ -42,17 +205,149 @@ parameters:
   default: 1
   options: [1, 2, 3, 4, 5, 6, 7, 8]
   hide: part
-- id: clock_source
-  label: Clock Source
-  dtype: string
-  options: ['', internal, external, gpsdo]
-  option_labels: [Default, Internal, External, O/B GPSDO]
-  hide: ${ 'none' if clock_source else 'part' }
-- id: time_source
-  label: Time Source
-  dtype: string
-  options: ['', internal, external, gpsdo]
-  option_labels: [Default, Internal, External, O/B GPSDO]
-  hide: ${ 'none' if time_source else 'part' }
+- id: clock_source_0
```

Comment:
We just recently removed all of these. I'm not necessarily against putting them back, but I wonder if we need to. Most of the time, we have the same time/clock source on all devices.

We recently added a patch to UHD to allow overriding individual devices' clock/time source with device args (e.g., `time_source=external,time_source0=internal`) but that's not yet part of a UHD release.

In any case, 'MIMO Cable' is not a valid option for RFNoC devices. It's only for USRP2/N210.

Reply:
Ok, then I will remove the mimo option. I do not like to shove everything into the device args. I understand that most of the time what happens, but I am not very happy to add yet another translation layer to try to make common tasks simpler. The problem is that it  MORE complicated,

Reply:
Have removed the MIMO options in a new commit.